ambari-web/app/controllers/wizard/step4_controller.js

```diff
@@ -335,12 +351,26 @@ App.WizardStep4Controller = Em.ArrayController.extend({
    */
   fileSystemServiceValidation: function(callback) {
     if(this.isDFSStack()){
+      const self = this;
       var primaryDFS = this.findProperty('isPrimaryDFS',true);
       if (primaryDFS) {
         var primaryDfsDisplayName = primaryDFS.get('displayNameOnSelectServicePage');
         var primaryDfsServiceName = primaryDFS.get('serviceName');
+        //if multiple DFS are not selected, remove the related error from the error array
+        let removeFsError = function () {
+          var fsError = self.get('errorStack').filterProperty('id',"multipleDFS");
```

Comment:
You should use findProperty because even when filterProperty returns empty array condition will pass Boolean([]) => true.

Reply:
This was the original code, I just put it in a function. But you are right.